start.sh

```diff
@@ -1,21 +1,19 @@
 #!/bin/bash
 # Locate JDK from $PATH
-path=`which java`
-# go through symlinks until we are at /usr/lib/jvm/...
-while [[ ${path} != /usr/lib/jvm/* ]]
-do
-path=`readlink $path`
-done
-while [[ $(dirname "$path") != /usr/lib/jvm ]]
-do
+path=`which javac`
+path=`readlink -f $path`
 path="$(dirname "$path")"
-done
-export JAVA_HOME=$path
+path="$path/.."
 
+TOOLS=$path/lib/tools.jar
+RSYNTAXTEXTAREA=$(echo $HOME/.m2/repository/com/fifesoft/rsyntaxtextarea/*/rsyntaxtextarea-*.jar)
+GSON=$(echo $HOME/.m2/repository/com/google/code/gson/gson/*/gson-*.jar)
+BYTEMAN=$(echo $HOME/.m2/repository/org/jboss/byteman/byteman-install/*/byteman-install-*.jar)
+JRD=$(echo $HOME/.m2/repository/java-runtime-decompiler/runtime-decompiler/*/runtime-decompiler-*.jar)
 # launch application
-java -cp ${JAVA_HOME}/lib/tools.jar:\
-$HOME/.m2/repository/com/fifesoft/rsyntaxtextarea/2.6.1/rsyntaxtextarea-2.6.1.jar:\
-$HOME/.m2/repository/io/github/soc/directories/10/directories-10.jar:\
```

Comment:
BYteman gone (likely correct) and directories added? Why so?

Reply:
Directories is no in pom anymore and byteman is there and it wasen't in the start.sh

Reply:
Yah, sorry. I ment vice versa.  Thank you. for clarification. I keep wondering, what is still used from byteman....